### PR TITLE
Suggested edits for README

### PR DESCRIPTION
**Pull request by drfleming0227**
@bluk @braebot 

Suggested edits for README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,152 +1,330 @@
...
+* For these checks to be valid, run them on a production system or one that *exactly* matches the production configuration.
+
+* * *
 
 ### Java
 
-* The TLS version can be set via [`SSLContext`](http://docs.oracle.com/javase/7/docs/api/javax/net/ssl/SSLContext.html).
-* **The latest Java (currently 8) is preferred.** In Java 8, TLSv1.2 is used by default when a TLS version is not specified.
+* [Java requirements](#java-requirements)
+* [To verify Java and TLS versions](#to-verify-java-and-tls-versions)
```
Comment (bluk) [inline README.md]: I think the link on line 39 isn't correct (think it should have `your` in it).

File: README.md [hunk 2]
```diff
@@ -1,152 +1,330 @@
...
 
-`$ ruby -r'net/http' -e 'puts Net::HTTP.get(URI("https://tlstest.paypal.com/"))'`
+* To update your dependencies, you might need to run `bundle update`.
 
-- On success, `PayPal_Connection_OK` will be printed.
-- On failure, a `OpenSSL::SSL::SSLError` or `EOFError` will be thrown.
+#### To verify Ruby and TLS versions
 
-### Node.js
+1. For the PayPal legacy Ruby SDK packaged as `PP_Ruby_NVP_SDK.zip`, download this [PP_Ruby_NVP_SDK.zip](https://github.com/paypal/TLS-update/blob/master/ruby/PP_Ruby_NVP_SDK.zip).
```
Comment (bluk) [inline README.md]: This is really a special callout that should probably be in its own section (e.g. something like `PayPal Legacy Ruby SDK Update`) and not part of the verify section.

File: README.md [hunk 3]
```diff
@@ -1,152 +1,330 @@
...
 
-You may find more official relevant information at the [PayPal Technical Support website](https://www.paypal-techsupport.com):
+As part of this requirement, PayPal and Braintree are making this upgrade alongside the rest of the payments industry. PayPal and Braintree are updating its services to require TLS 1.2 for all HTTPS connections. PayPal and Braintree will also require HTTP/1.1 for all connections.
+
+For more official, relevant information, see the [PayPal Technical Support website](https://www.paypal-techsupport.com):
 * [TLS 1.2 and HTTP/1.1 Upgrade Microsite](https://www.paypal-knowledge.com/infocenter/index?page=content&id=FAQ1914&expand=true&locale=en_US)
-* [2016 Merchant Security Roadmap Microsite](https://www.paypal-knowledge.com/infocenter/index?page=content&id=FAQ1913&expand=true&locale=en_US)
+* [2016-2017 Merchant Security Roadmap Microsite](https://www.paypal-knowledge.com/infocenter/index?page=content&id=FAQ1913&expand=true&locale=en_US)
 * [SSL Certificate Upgrade Microsite](https://www.paypal-knowledge.com/infocenter/index?page=content&id=FAQ1766&expand=true&locale=en_US)
```
Comment (bluk) [inline README.md]: Do you think we should add a link to the [Braintree blog post](https://www.braintreepayments.com/blog/updating-your-production-environment-to-support-tlsv1-2/) as well in a different section?

File: README.md [hunk 4]
```diff
@@ -1,152 +1,330 @@
...
-- On failure, an `URLError` will be raised: <br/>
-`urllib2.URLError: <urlopen error EOF occurred in violation of protocol (_ssl.c:590)>`
-`urllib2.URLError: <urlopen error [Errno 54] Connection reset by peer>`
+* * *
 
 ### Ruby
 
-Ruby 2.0.0 or above is required to use TLSv1.2 from the system supplied OpenSSL. TLSv1.2 requires OpenSSL 1.0.1c or higher. Therefore, both `Ruby > 2.0.0` and `OpenSSL > 1.0.1c` are required. You may also need to run `bundle update` to update your dependencies.
+* [Ruby requirements](#ruby-requirements)
+* [To verify your Ruby and TLS versions](#to-verify-your-ruby-and-tls-versions)
```
Comment (bluk) [inline README.md]: The `your` here should match line 249.

File: README.md [hunk 5]
```diff
@@ -1,152 +1,330 @@
...
-- On success, 200 is printed.
-- On failure, a network error is printed.
+    * On failure, a `OpenSSL::SSL::SSLError` or `EOFError` is thrown.
+
+* * *
+
+### Node
+
+* [Node requirements](#node-requirements)
+* [To verify your Node and TLS versions](#to-verify-your-node-and-tls-versions)
```
Comment (bluk) [inline README.md]: `your` link should match the title on line 279.

**Review by bluk — CHANGES_REQUESTED** [review]
Mostly minor updates. Thanks!

**Comment by drfleming0227** [comment]
@bluk Incorporated your comments https://github.com/paypal/TLS-update/pull/22/commits/85202614ef546fb41ba7099be75a2681526fe4eb - thanks!

**Review by bluk — APPROVED** [review]

**Review by nadhlad — APPROVED** [review]

**Pull request merged**